src/main/rx/fport.c

```diff
@@ -390,6 +397,13 @@ static bool fportProcessFrame(const rxRuntimeState_t *rxRuntimeState)
     return true;
 }
 
+static timeUs_t fportFrameTimeUs(void)
+{
+    const timeUs_t result = lastRcFrameTimeUs;
+    lastRcFrameTimeUs = 0;
```

Comment:
This is not needed.

Reply:
Yes it is. This is part of the logic for all the protocols. Each time the RX task retrieves the last frame we reset it so that we can tell that no more RC channels frame have been received.

Reply:
Can we please rename this to something less misleading that does not insinuate that this is the interval between received frames, like 'fportFakeFrameTimeUs()`? Adding more of these bogus 'measurements' will just require more work to rename or remove them all when an actual measurement is added.

Reply:
It's already named appropriately. last***Rc***FrameTimeUs. The entire goal is to capture the time of the last valid RC channels frame. We don't care about other frames like telemetry, etc. So it's not, and never has been about capturing the time of the last protocol frame.

Reply:
Ok, I will open a pull request to add **RC frame interval** measuerements that do not rely on adding in bogus data by 'invalidating' the reality based on some factors that are entirely unrelated to RC frames being received. Then we can use that.

Reply:
The "reality" is that we need to know the inter-frame timing between valid, complete, RC channels data frames. We also need to know when we've stopped receiving said frames. We don't care about how long ago the last good frame was if they've stopped. We have no use for this information. The code as it exists meets these needs.

Reply:
Not sure we are talking about the same thing when we say 'reality' - I am talking about actual, measurable facts, and you seem to be talking about some concept of what we want to achieve.
I am not saying that what you are trying to achieve is wrong, what I am saying is that it is wrong to hack the data measurement at the lowest possible level in order to make 'measurements' fit what you want to achieve.

Reply:
Why is "reality" some abstract that's no use to us? The reality we want is to know the time since the last valid RC channels frame and/or that no frame has been received since the last valid one. That what the code does. Resetting `lastRcFrameTimeUs = 0` after reading serves as an indicator that no more frames have been received since the last. Again, if we've stopped receiving data frames we don't care _when_ the last valid one was. It's of no value to us, so why should we code it in a way that captures it (then have to try and extract the "no longer receiving data" from this ever-going interval).